apps/datahub/src/app/home/navigation-menu/navigation-menu.component.html

```diff
@@ -1,7 +1,12 @@
 <button
   *ngFor="let l of tabLinks"
   [routerLink]="l.link"
-  class="uppercase truncate btn-text-white cursor-pointer hidden sm:block"
+  [ngClass]="
+    (activeLink$ | async).link === l.link
+      ? 'decoration-primary'
+      : 'decoration-transparent hover:decoration-primary transition-colors'
+  "
+  class="hidden sm:block sm:py-4 uppercase truncate text-white underline decoration-4 underline-offset-[19px]"
```

Comment:
There is no effect on the text anymore when hovering, just the bar below. I guess this is on purpose?

Reply:
Yes, everything i have done have been asked by lucile btw